openapi/v1/core-data.yaml

```diff
@@ -1030,6 +1036,18 @@ components:
           type: string
       description: Core and MetaData value descriptor that describes
         device/sensor data sent and received
+    tagsKeyValue:
```

Comment:
we don't need to define `tagsKeyValue` object, because it would become.
[{"key": "gateway-id", "value": "HoustonStore-000123"}, {"key": "OOO", "value": "XXX"}]

Reply:
Yep, yo are correct!! Good catch!

I thought this is required for the data to be un-marshaled correctly into the `map[string]string`, but per my test below it is a simple object with the the keys as properties and the values as the property's values.

I add this to V1's event.go just prior to persisting the Event and then queered the events.
```
	e.Tags = make(map[string]string)
	e.Tags["gateway-id"] = "HoustonStore-000123"
	e.Tags["OOO"] = "YYY"
```
and the resulting JSON is this:
```
    {
        "id": "a1abc227-f17d-4c74-957e-523d57d6ac2a",
        "device": "Random-Boolean-Device",
        "created": 1599066555273,
        "origin": 1599066555271553579,
        "readings": [
            {
                "id": "0230e6ad-35ba-4af1-a724-f80e62a781b1",
                "created": 1599066555273,
                "origin": 1599066555271455776,
                "device": "Random-Boolean-Device",
                "name": "Bool",
                "value": "true",
                "valueType": "Bool"
            }
        ],
        "tags": {
            "OOO": "YYY",
            "gateway-id": "HoustonStore-000123"
        }
```

I will use this as the example. :-)

Reply:
Corrected.